src/components/YourCompany/OfficeAddresses.vue

```diff
@@ -137,63 +135,65 @@
             </div>
-          </div>
-        </li>
-
-        <!-- Registered Delivery Address -->
-        <li class="ma-5">
-          <div class="meta-container">
-            <label>Delivery Address</label>
-            <div class="meta-container__inner">
-              <div class="form__row">
-                <v-checkbox
-                  id="registered-mailing-same-chkbx"
-                  class="inherit-checkbox"
-                  label="Same as Mailing Address"
-                  hide-details
-                  v-model="inheritMailingAddress"
-                  @change="setDeliveryAddressToMailingAddress()"
-                />
-              </div>
-              <div
-                class="address-wrapper pt-6"
-                v-if="!isSame(mailingAddress, deliveryAddress, 'actions') || !inheritMailingAddress"
-              >
-                <base-address ref="regDeliveryAddress"
-                  id="address-registered-delivery"
-                  v-if="!inheritMailingAddress"
-                  :address="deliveryAddress"
-                  :editing="true"
-                  :schema="addressSchema"
-                  @update:address="updateAddress(AddressTypes.DELIVERY_ADDRESS, deliveryAddress, $event)"
-                  @valid="updateValidity(AddressTypes.DELIVERY_ADDRESS, $event)"
-                />
+          </li>
+
+          <!-- Registered Delivery Address -->
+          <li class="ma-5">
+            <div class="meta-container">
+              <label>Delivery Address</label>
+              <div class="meta-container__inner">
+                <div class="form__row">
+                  <v-checkbox
+                    id="registered-mailing-same-chkbx"
+                    class="inherit-checkbox"
+                    label="Same as Mailing Address"
+                    hide-details
+                    v-model="inheritMailingAddress"
+                    @change="setDeliveryAddressToMailingAddress()"
+                  />
+                </div>
+                <div
+                  class="address-wrapper pt-6"
+                  v-if="!isSame(mailingAddress, deliveryAddress, 'actions') || !inheritMailingAddress"
+                >
+                  <base-address ref="regDeliveryAddress"
+                    id="address-registered-delivery"
+                    v-if="!inheritMailingAddress"
+                    :address="deliveryAddress"
+                    :editing="true"
+                    :schema="addressSchema"
+                    @update:address="updateAddress(AddressTypes.DELIVERY_ADDRESS, deliveryAddress, $event)"
+                    @valid="updateValidity(AddressTypes.DELIVERY_ADDRESS, $event)"
+                  />
+                </div>
               </div>
             </div>
-          </div>
-        </li>
+          </li>
+        </div>
 
-        <template v-if="entityFilter(EntityTypes.BCOMP)">
+        <div id="edit-records-address" v-if="entityFilter(EntityTypes.BCOMP)">
```

Comment:
If you see a large diff here, click on the gear icon at the top of this page and hide whitespace changes.